<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BINSInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget
{
	background-color: rgb(253, 253, 253);
}

QTabBar::tab
{
	min-width:264px;
	height:36px;
}

QGroupBox
{
	border:1px solid rgb(112,112,112);
	border-radius:4px;
}

QTabWidget
{
	
	background-color: rgb(245, 245, 245);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QTabWidget" name="mainTabWidget">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabInfoAndSetting">
         <attribute name="title">
          <string>Информация и настройки</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>26</number>
          </property>
          <property name="leftMargin">
           <number>21</number>
          </property>
          <property name="rightMargin">
           <number>21</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="layoutInfoAndSetting">
            <property name="spacing">
             <number>20</number>
            </property>
            <item alignment="Qt::AlignTop">
             <widget class="BINSParamUI" name="widgetBINSParam" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="InfoFromBINSUI" name="widgetBINSInfo" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabControl">
         <attribute name="title">
          <string>Управление</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>21</number>
          </property>
          <property name="rightMargin">
           <number>21</number>
          </property>
          <property name="verticalSpacing">
           <number>26</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0" alignment="Qt::AlignTop">
             <widget class="ConnectionWithBINSUI" name="widgetConnState" native="true"/>
            </item>
            <item row="1" column="0" alignment="Qt::AlignTop">
             <widget class="CorrectionControlUI" name="widgetCorrControl" native="true"/>
            </item>
            <item row="0" column="0" alignment="Qt::AlignTop">
             <widget class="BINSRemoteControlUI" name="widgetRemoteControl" native="true"/>
            </item>
            <item row="2" column="0" alignment="Qt::AlignTop">
             <widget class="RecControlUI" name="widgetRecControl" native="true"/>
            </item>
            <item row="0" column="1" rowspan="4">
             <layout class="QVBoxLayout" name="layoutOutsideSystem">
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Информация от внешних систем</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignTop">
               <widget class="QGroupBox" name="groupOutsideData">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="topMargin">
                  <number>13</number>
                 </property>
                 <property name="bottomMargin">
                  <number>13</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="OutsideSystemInfo" name="widgetOutsideData" native="true"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BINSParamUI</class>
   <extends>QWidget</extends>
   <header location="global">binsparamui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoFromBINSUI</class>
   <extends>QWidget</extends>
   <header location="global">infofrombinsui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BINSRemoteControlUI</class>
   <extends>QWidget</extends>
   <header>binsremotecontrolui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CorrectionControlUI</class>
   <extends>QWidget</extends>
   <header>correctioncontrolui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecControlUI</class>
   <extends>QWidget</extends>
   <header>reccontrolui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionWithBINSUI</class>
   <extends>QWidget</extends>
   <header>connectionwithbinsui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OutsideSystemInfo</class>
   <extends>QWidget</extends>
   <header>outsidesysteminfo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
